assets/js/rivet-docs.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (C) 2018 The Trustees of Indiana University
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import { createCopyright, setIndeterminate, stickySupportForm, copyButtonConfirm, analyticsTracking } from './plugins';
+
+/**
+ * One-off scripts for random sutff. See 'plugins.js'
+ */
+
+// Third-party libraries
```

Comment:
We can get rid of this extra comment. The one above it seems descriptive enough. In the comment that stays can you update _sutff_ to _stuff_. One of my many typos that's scattered around. ☹️